common/dev/ina233.c

```diff
@@ -17,25 +17,43 @@
 #include <stdio.h>
 #include <stdint.h>
 #include <string.h>
+#include <logging/log.h>
 #include "sensor.h"
 #include "pmbus.h"
 #include "hal_i2c.h"
 
+LOG_MODULE_REGISTER(dev_ina233);
+
+#define INA233_CALIBRATION_OFFSET 0xD4
+
 uint8_t ina233_read(uint8_t sensor_num, int *reading)
 {
 	if (reading == NULL) {
-		printf("[%s] input parameter reading is NULL\n", __func__);
+		LOG_ERR("read func: input parameter reading is NULL\n");
```

Comment:
The LOG_XXX functions automatically add a newline so it can be removed from the log message.

```
LOG_ERR("read func: input parameter reading is NULL");
```

Remove "read func: " (is included by logging service if enabled.)